moveit_demo_nodes/run_moveit_cpp/config/panda.urdf.xacro

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0"?>
+<robot xmlns:xacro="http://www.ros.org/wiki/xacro" name="panda">
+
+    <!-- Import panda urdf file -->
+    <xacro:include filename="$(find moveit_resources_panda_description)/urdf/panda.urdf" />
+
+    <!-- Import panda ros2_control description -->
+    <xacro:include filename="panda.ros2_control.xacro" />
+    <xacro:include filename="panda_hand.ros2_control.xacro" />
+
+    <xacro:panda_ros2_control name="PandaFakeSystem" />
```

Comment:
So, I assume this is optional... What happens when you want to run the driver? Adding face controller parameters to the description somehow feels strange

Reply:
I'd suggest injecting either the fake system of the non-fake one in the launch file or via a xacro argument, they should be considered mutually exclusive